nakadi-java-client/src/main/java/nakadi/OkHttpResource.java

```diff
@@ -198,7 +197,8 @@ private Request.Builder prepareBuilder(String method, String url, ResourceOption
     if (body != null) {
       {
         RequestBody requestBody =
-            RequestBody.create(MediaType.parse(APPLICATION_JSON_CHARSET_UTF8), body.content());
+                RequestBody.create(MediaType.parse((String) options.headers().get("Content-Type")), body.content());
```

Comment:
what if Content-Type is not there? 
can you replace Content-Type with the variable from okhttp (if any)?